static/app/components/duration/duration.tsx

```diff
@@ -0,0 +1,45 @@
+import type {HTMLAttributes} from 'react';
+import styled from '@emotion/styled';
+
+import formatDuration, {type Format} from 'sentry/utils/duration/formatDuration';
+import type {Duration as TDuration, Unit} from 'sentry/utils/duration/types';
+
+interface Props extends HTMLAttributes<HTMLTimeElement> {
+  /**
+   * The Duration that you want to render
+   */
+  duration: TDuration;
+
+  /**
+   * How granular to render the value. You can pass in something that has `ms`
+   * precision but only show the total number of seconds (ala Math.floor)
```

Comment:
```suggestion
   * precision but only show the total number of seconds (aka Math.floor)
```

Reply:
ah, i lost a fix for this. updated again.